actioncable/lib/assets/javascripts/action_cable/connection.coffee

```diff
@@ -75,7 +79,17 @@ class ActionCable.Connection
     error: ->
       @disconnect()
 
+    dispatchEvent: (identifier, type, message) ->
+      unless identifier == "_ping"
+        event = document.createEvent('Event')
+        event.initEvent("cable:#{type}", true, false)
+        channel = JSON.parse(identifier).channel
+        event.cable = {data: message, identifier: channel}
+        document.dispatchEvent(event)
```

Comment:
Users only ever interface with channels, identifiers is a private construct. You can see the mental model break down where extracting a channel from an identifier and then calling it an identifier in the object.

Should switch to calling it channel.